app/views/shelter_pets/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Available Pets</h1>
+
+<% @pets.each do |pet| %>
+<img src = "<%= pet.image %>" alt:="<%= pet.name %>" </br>
+<h2><%= link_to "#{pet.name}", "/pets/#{pet.id}" %> </h2>
+<p> Age: <%= pet.age %> </p>
+<p> Sex: <%= pet.sex %> </p>
+<% end %>
+
+
+<a href= "/shelters/<%=@shelter.id%>/pets/new">Create Pet</a>
```

Comment:
Note to self: the @shelter.id _has_ to be wrapped in the erb tag or the link won't work